test/functional/feature_rbf.py

```diff
@@ -78,10 +81,7 @@ def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
```

Comment:
Did you forget to remove this now that the test can be run without the wallet compiled?

Reply:
Can it?

Reply:
Nope 👍